README.md

```diff
@@ -183,6 +183,7 @@ The number after the course name stands for the year in which the course was mad
 - [Artificial intelligence for robotics](https://www.udacity.com/course/artificial-intelligence-for-robotics--cs373)
 - [Neural networks for machine learning](https://www.coursera.org/learn/neural-networks)
 - [Machine learning in Python with scikit-learn](https://github.com/justmarkham/scikit-learn-videos#readme)
+- [mlcourse.ai](http://mlcourse.ai) - open Machine Learning course by OpenDataScience
```

Comment:
Uppercase open.

Also, no need to create new PR, just add a commit and we merge this.

Reply:
Also it needs to end in full stop.